src/main/migrations/changesets/change_execution_unique_constraint.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="tjeandet" id="change_execution_unique_constraint">
+        <comment>
+            Change unique constraint for Execution Table to include INDEX column.
+        </comment>
+        <dropUniqueConstraint
+                tableName="EXECUTION"
+                constraintName="UK_EX_WORKFLOW_EXECUTION_ID" />
+        <addUniqueConstraint
+                tableName="EXECUTION"
+                columnNames="WORKFLOW_EXECUTION_ID, CALL_FQN, INDEX"
+                constraintName="UK_EX_WORKFLOW_EXECUTION_ID"/>
```

Comment:
Though we're currently using MySQL, that identifier tragically would not work on Oracle as it is longer than 30 characters.  Given that other people are starting to look at using Cromwell in ways we hadn't anticipated, I'd prefer not to choose a name that we know wouldn't work on a popular RDBMS.  Which reminds me we still need to fix the name of the INDEX column... :frowning: 

Maybe `UK_WORKFLOW_CALL_INDEX`?